tools/backups2datalad/adataset.py

```diff
@@ -242,12 +242,20 @@ async def remove(self, path: str) -> None:
             lockfile = self.pathobj / ".git" / "index.lock"
             output = e.stdout.decode("utf-8")
             if lockfile.exists() and str(lockfile) in output:
-                r = await aruncmd("fuser", "-v", lockfile, stdout=subprocess.PIPE)
+                r = await aruncmd(
+                    "fuser",
+                    "-v",
+                    lockfile,
+                    stdout=subprocess.PIPE,
+                    stderr=subprocess.STDOUT,
```

Comment:
just a note, correct me if I am wrong - we will redirect stderr to stdout and that one is displayed below

Reply:
stdout and stderr are combined, and the combination is displayed.